Sources/SwiftProtobuf/Message+BinaryAdditions_Data.swift

```diff
@@ -31,14 +31,75 @@ extension Message {
   ///   - options: The ``BinaryDecodingOptions`` to use.
   /// - Throws: ``BinaryDecodingError`` if decoding fails.
   @inlinable
+  @available(*, deprecated, renamed: "init(contiguousBytes:extensions:partial:options:)")
```

Comment:
I guess these might cause a lot of warnings for clients?  Should we do that one the first move to head, or wait one release to add these just to keep the noise of this migration down a little?

Reply:
If we don't warn, users won't know we're planning on deprecating this and will not move. I think it's better to just get this out of the way now instead of having a couple of releases that may both cause some friction - I asked around the team and we all seem to agree on this.

Reply:
Ugh, I didn't think about this before.  We've broken the api symmetry.

The original api design for all three formats was to have naming consistency between parsing and generation.  For the Binary format, we settled on _serialized_.  So with this change, generation will be `serializedBytes`, but parsing will be `continuousBytes`. 😞

Is there no way (with overloads, etc.) that we can keep the original naming symmetry to make such a common api easier to use?

Reply:
@tbkka fyi

Reply:
Ah, that's frustrating (but a fair point). I've worked around this by adding some new overloads - would you mind taking a look at the last commit and seeing if you agree with the workaround? If so I can regenerate the protos.